cibuild.cmd

```diff
@@ -11,19 +14,27 @@ if /I "%1" == "/debug" set BuildConfiguration=Debug&&shift&& goto :ParseArgument
 if /I "%1" == "/release" set BuildConfiguration=Release&&shift&& goto :ParseArguments
 if /I "%1" == "/test32" set Test64=false&&shift&& goto :ParseArguments
 if /I "%1" == "/test64" set Test64=true&&shift&& goto :ParseArguments
+if /I "%1" == "/restore" set BuildRestore=true&&shift&& goto :ParseArguments
 call :Usage && exit /b 1
 :DoneParsing
 
 call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat"
 
-REM Build the compiler so we can self host it for the full build
-nuget.exe restore -nocache -verbosity quiet %RoslynRoot%build/ToolsetPackages/project.json
-nuget.exe restore -nocache -verbosity quiet %RoslynRoot%build/Toolset.sln
+REM Restore the NuGet packages 
+if "%BuildRestore%" == "true" (
+    nuget.exe restore -nocache -verbosity quiet %RoslynRoot%build/ToolsetPackages/project.json
+    nuget.exe restore -nocache -verbosity quiet %RoslynRoot%build/Toolset.sln
+    nuget.exe restore -nocache %RoslynRoot%build\ToolsetPackages\project.json
```

Comment:
Ha.  I just copied what was already there.  Didn't think to look at it.  But yes looks like a duplicate.